test/NCalc.Benchmarks/EvaluateVsLambdaBenchmark.cs

```diff
@@ -38,13 +36,13 @@ public bool Evaluate()
     {
         return (bool)Expression.Evaluate()!;
     }
-
+    
```

Comment:
Unfortunately, spaces are still added on empty lines...

Reply:
I forgot to pull master before creating this branch, I will format this file again